### PR TITLE
Updates for alpha.9

### PR DESCRIPTION
**Pull request by martincostello**
- Update CHANGELOG.
- Bump samples to 8.0.0-alpha.9.

### CODE REVIEW

File: samples/Extensibility/Program.cs [hunk 1]
```diff
@@ -23,9 +23,9 @@
 // SIMPLE EXTENSIBILITY MODEL (INLINE STRATEGY)
 // ------------------------------------------------------------------------
 
-strategy = new CompositeStrategyBuilder()
+strategy = new ResiliencePipelineBuilder()
     // Just add the strategy instance directly
-    .AddStrategy(new MySimpleStrategy())
+    .AddStrategy(_ => new MySimpleStrategy())
```
Comment (martincostello) [inline samples/Extensibility/Program.cs:28]: @martintmk This isn't compiling and I wasn't sure what the intended fix was supposed to be.

Comment (martintmk) [reply samples/Extensibility/Program.cs:28]: We can do:

``` csharp
.AddStrategy(_ => new MySimpleStrategy(), new MySimpleStrategyOptions())
```

Comment (martintmk) [reply samples/Extensibility/Program.cs:28]: Let's create empty `MySimpleStrategyOptions`

File: samples/Extensibility/Program.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@
 // ------------------------------------------------------------------------
 // Usage of custom strategy
 // ------------------------------------------------------------------------
-var strategy = new CompositeStrategyBuilder()
+var strategy = new ResiliencePipelineBuilder()
```
Comment (martintmk) [inline samples/Extensibility/Program.cs:7]:
```suggestion
var pipeline = new ResiliencePipelineBuilder()
```

**Review by martintmk — APPROVED** [review]

**Pull request merged**